manual/python/index.md

```diff
@@ -482,7 +482,7 @@ TypeError: CppBase not an acceptable base: no virtual destructor
 | cppyy.gbl.nullptr                         | cppyy.nullptr                   |
 | cppyy.gbl.PyROOT.TPyException             | cppyy.gbl.CPyCppyy.TPyException |
 | buffer.SetSize(N)                         | buffer.reshape((N,))            |
-| obj.\_\_cppname\_\_                       | obj.\_\_cpp\_name\_\_           |
+| obj.\_\_cppname\_\_                       | klass.\_\_cpp\_name\_\_         |
```

Comment:
```suggestion
| obj.\_\_cppname\_\_                       | type(obj).\_\_cpp\_name\_\_         |
```
?

Just a suggestion, I'm never sure what `klass` stands for

Reply:
That's better, thanks @eguiraud !